ee/modules/015-admission-policy-engine/images/trivy-provider/main.go

```diff
@@ -42,13 +45,32 @@ func main() {
 
 	zapLog, err := zap.NewDevelopment(zap.AddStacktrace(zap.ErrorLevel))
 	if err != nil {
-		panic(fmt.Sprintf("unable to initialize logger: %v", err))
+		zapLog.Fatal(fmt.Sprintf("unable to initialize logger: %v", err))
 	}
 	logger := zapr.NewLogger(zapLog).WithName("trivy-provider")
 
+	if err = initJavaDB(); err != nil {
+		zapLog.Fatal(fmt.Sprintf("Couldn't initialize JavaDB: %v", err))
+	}
+	zapLog.Info("JavaDB was successfully initialized")
+
+	go func() {
+		ticker := time.NewTicker(1 * time.Hour)
+		defer ticker.Stop()
```

Comment:
It feels like this defer will never be executed 😆

Reply:
better?